src/lib/components/TabsMenu.svelte

```diff
@@ -4,12 +4,16 @@ A component to display tabbed menus.
 -->
 <script lang="ts">
     import { createEventDispatcher } from 'svelte';
+    import { globalConfig } from '$lib/data/stores';
 
     export let options: App.TabMenuOptions = { '': { component: '', props: {} } };
     export let active = '';
-    export let _class = 'bg-primary';
     const dispatch = createEventDispatcher();
 
+    $: primaryColor = $globalConfig.themes
+        .find((x) => x.name === 'Normal') // TODO: change to fetch the current theme
+        .colorSets.find((x) => x.type === 'main').colors['PrimaryColor'];
```

Comment:
'PrimaryColor' should be changed to a style lookup.  Contact @davidmoore1 to find out the style name.